app/models/account/transfer.rb

```diff
@@ -72,27 +72,23 @@ def single_currency_transfer?
 
     def transaction_count
       unless entries.size == 2
-        # i18n-tasks-use t('activerecord.errors.models.account/transfer.attributes.entries.must_have_exactly_2_entries')
```

Comment:
These introduce too much maintenance overhead at this stage.  Removing and ignoring.